data/common.yaml

```diff
@@ -19,3 +19,4 @@ superset::concurrency: 4
 superset::manage_database: true
 superset::ldap_enabled: false
 superset::row_limit: None
+superset::dynamic_plugins: False
```

Comment:
Change to:
`superset::dynamic_plugins: false`
to reflect boolean rather than string type